doc/source/development.rst

```diff
@@ -1,8 +1,130 @@
+do not underestimate the power of OpenUpgrade.
+The migration of the account module to version 13 is the kind of situation where
+the full power of OpenUpgrade was unleashed to successfully overcome the
+"acc-pocalypse":
+https://github.com/OCA/OpenUpgrade/pull/2275/files.
+
+
+Learn from existing migration scrips
+------------------------------------
+
+
+Since version 14, the migration scripts are located in:
+``OpenUpgrade/openupgrade-framework/openupgrade_scripts/scripts/<module-name>``
+
+During the review for a given module, you can follow this process:
+
+- Review the analysis (read more in :doc:`analysis`). First the file
+  `openupgrade_analysis.txt` showing some differences between the 2 versions of
+  the module. Then, `openupgrade_analysis_work.txt` to see what the developer
+  planned to do.
+- Then you can check the other files to see how the plan was achieved.
+
+You should pick modules with which you are familiar with in both version and for which
+you are aware of the changes.
+
+Learn from code review of open PRs
+----------------------------------
+
+https://github.com/OCA/OpenUpgrade/pulls
+
+This will engage you in a discussion with other contributors and help you understand
+how developers selected one way or another to implement the migration scripts.
+
+The Trial and error process for the development of your scripts
+---------------------------------------------------------------
+
+Basically, this is the happening during the step when you try to run the upgrade
+described in :doc:`migration_details`:
+
+- [A] get the copy of your database in old version as `db-upgrade`, it is
+  easy to do through the database manager of your old odoo
+  `http://localhost:8012/web/database/manager`
+- [B] run `odoo -d db-upgrade -u all --stop-after-init`
+- [C] In case of error, fix the error (adding or editing migration
+  scripts), then rerun **without** the `--upgrade all`:
+  `odoo -d db-upgrade --stop-after-init`. This will continue the upgrade
+  process from where it left off, starting with the module that caused the
+  error. That way, the migration will not run the end-migration scripts of
+  the modules that were already upgraded before the error, so in case of a
+  live migration you always need to restore the database. But running this way
+  will help you save a lot of time in the upgrade process.
+
+Note that for the time being [C] is not available in all versions
+(see explanations here: https://github.com/OCA/OpenUpgrade/pull/2499).
+It is available for 12 and 13:
+https://github.com/OCA/OpenUpgrade/pulls?q=is%3Apr+%5BFIX%5D+reset+exception
+
+Whenever facing unexpected errors, you might want to restart from [A] as
+the step [C] might leave your database in an inconsistent state.
+
+Restart the upgrade of the failed module instead of upgrading all systematically
+................................................................................
+
+**As an alternative to the step [C] mentioned above...**
```

Comment:
I think the other method should be the alternative method, and this one should be the usual method, because the above method only applies to v12 and v13, and this are the docs for v14 , and v14 doesn't have enabled that way at this moment. Could you change the order?

Reply:
ok, I will do this change of order.

Reply:
it's done @MiquelRForgeFlow